test/refactoring/rename/shorthand_shadow.swift

```diff
@@ -1,16 +1,11 @@
-// Local rename starts from the `DeclContext` of the renamed `Decl`. For
-// closures that means we have no parents, so check that case specifically.
-
-func renameInClosure() {
-  // RUN: %refactor -rename -dump-text -source-filename %s -pos=%(line+2):10 -new-name=renamed | %FileCheck %s
-  // CHECK: shorthand_shadow.swift [[# @LINE+1]]:10 -> [[# @LINE+1]]:18
-  _ = { (toRename: Int?) in
-    // RUN: %refactor -rename -dump-text -source-filename %s -pos=%(line+2):12 -new-name=renamed | %FileCheck %s
-    // CHECK: shorthand_shadow.swift [[# @LINE+1]]:12 -> [[# @LINE+1]]:20
-    if let toRename {
-      // RUN: %refactor -rename -dump-text -source-filename %s -pos=%(line+2):11 -new-name=renamed | %FileCheck %s
-      // CHECK: shorthand_shadow.swift [[# @LINE+1]]:11 -> [[# @LINE+1]]:19
-      _ = toRename
-    }
+// RUN: %refactor -rename -dump-text -source-filename %s -pos=%(line+2):29 -new-name=renamed | %FileCheck %s --check-prefix=OPTIONAL
+// OPTIONAL: shorthand_shadow.swift [[# @LINE+1]]:29 -> [[# @LINE+1]]:32
```

Comment:
I think it might be worth keeping a version of this test that is inside a closure. At least in solver-based land they still behave differently than top-level code.

Reply:
I just moved that to the sourcekitd-test. This one now just checks we handle shorthand shadow correctly from refactor as well.

Reply:
...oh I forgot to add that file :)